web/router.ex

```diff
@@ -20,6 +22,15 @@ defmodule CoursePlanner.Router do
     plug Coherence.Authentication.Session, protected: true
   end
 
+  pipeline :protected_api do
+    plug :accepts, ["json"]
+    plug :fetch_session
+    plug :fetch_flash
```

Comment:
I think we don't need the plugs :fetch_flash, :protect_from_forgery, and :put_secure_browser_headers for this pipeline, could you check it please?